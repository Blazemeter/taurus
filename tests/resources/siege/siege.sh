#!/bin/sh -xe
echo -e "[0;34m   1,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:14[0m"
echo -e "[0;34m   2,HTTP/1.1,200,  0.33,   3372,/,0,2015-12-20 21:30:15[0m"
