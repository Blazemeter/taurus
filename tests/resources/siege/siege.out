[0;34m   4,HTTP/1.1,200,  0.36,   3372,/,0,2015-12-20 21:30:13[0m
[0;34m   0,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-20 21:30:13[0m
[0;34m   1,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-20 21:30:14[0m
[0;34m   3,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:31:14[0m
[0;34m   2,HTTP/1.1,200,  0.40,   3372,/,0,2015-12-20 21:34:14[0m
[0;34m   0,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-21 21:33:14[0m
[0;34m   4,HTTP/1.1,200,  0.41,   3372,/,0,2015-12-22 21:30:14[0m
[0;34m   1,HTTP/1.1,200,  0.40,   3372,/,0,2015-12-20 21:30:15[0m
[0;34m   3,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:15[0m
[0;34m   2,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:37:15[0m
